setup.cfg

```diff
@@ -49,11 +49,6 @@ install_requires =
 [options.packages.find]
 where = src
 
-[options.entry_points]
-console_scripts =
-	python3 = spetlr.alias:python3
-	spetlr_task = spetlr.entry_points.generalized_task_entry_point:main
```

Comment:
We probably still need setlr_task entry point

Reply:
The spetlr_task entry point is being used all over in our solution

Reply:
@tbg-delegate
What is you input here. This caused an issue with some ml related stuff, you are working with. Lets see if we can find a solution togheder.

Reply:
It's fine, if we need the endpoint. But we shouldn't receive an error with "too many symbolic links" when installing on Ubuntu and running python code. So the bugfix/change should instead handle that problem, instead of removing the entry point.

According to the error messages, I understood the problem as simply pointing to an incorrect python version installed, rather than the one with spetlr installed. I could be wrong.

My personal opinion is that "overwriting" the python3 system endpoint using a python package is a bit dangerous design, but I will live with it, if we cannot find another solution.

Reply:
@tbg-delegate I originally implemented this hack. I agree that it is not great. The idea was not to overwrite the system python because in all out solutions we use a venv. I understand that your solution doesn't. We are working on a way to make the tests work without this fix. As a fallback workaround we could build a separate pypi package that implements exactly this one entry_point and we can install that to make our tests work an you can leave it out if it bothers you.